### PR TITLE
Feature/priority update

### PR DESCRIPTION
**Pull request by jeffvoight**

### CODE REVIEW

File: gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java [hunk 1]
```diff
@@ -34,13 +34,14 @@
...
   name = "File names should comply with a naming convention",
-  priority = Priority.MINOR,
+  priority = Priority.CRITICAL,
   tags = {Tags.CONVENTION})
 @SqaleConstantRemediation("5min")
 @ActivatedByDefault
 public class FileNameCheck extends DoubleDispatchVisitorCheck {
 
-  private static final String DEFAULT = "^[a-z][-A-Za-z0-9]*\\.feature$";
+  //private static final String DEFAULT = "^[a-z][-A-Za-z0-9]*\\.feature$";
```
Comment (jonaqorri) [inline gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java]: remove commented code

File: gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java [hunk 2]
```diff
@@ -34,13 +34,14 @@
...
+  private static final String DEFAULT = "^ac-\\d{5,6}\\.feature$";
```
Comment (jonaqorri) [inline gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java]: \\d{1, }

File: gherkin-checks/src/main/java/org/sonar/gherkin/checks/OneSingleThenPerScenarioCheck.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import org.sonar.plugins.gherkin.api.visitors.issue.PreciseIssue;
+import org.sonar.squidbridge.annotations.ActivatedByDefault;
+import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Rule(
+  key = "one-single-then-per-scenario",
+  name = "There should be one single Then step per scenario",
```
Comment (jonaqorri) [inline gherkin-checks/src/main/java/org/sonar/gherkin/checks/OneSingleThenPerScenarioCheck.java:38]: This is not in the major section of the Gherkin SonarQube Rules word doc you sent me....does it need to be added?

File: gherkin-checks/src/test/java/org/sonar/gherkin/checks/FileNameCheckTest.java [hunk 4]
```diff
@@ -30,14 +30,14 @@
...
-    GherkinCheckVerifier.issues(check, CheckTestUtils.getTestFile("file-name/file-name-ok.feature"))
+    GherkinCheckVerifier.issues(check, CheckTestUtils.getTestFile("file-name/ac-11111.feature"))
       .noMore();
   }
 
   @Test
   public void should_not_follow_the_default_naming_convention_and_raise_an_issue() {
     GherkinCheckVerifier.issues(check, CheckTestUtils.getTestFile("file-name/FileNameKO.feature"))
-      .next().withMessage("Rename this file to match the regular expression: ^[a-z][-A-Za-z0-9]*\\.feature$")
+      .next().withMessage("Rename this file to match the regular expression: ^ac-\\d{5,6}\\.feature$")
```
Comment (jonaqorri) [inline gherkin-checks/src/test/java/org/sonar/gherkin/checks/FileNameCheckTest.java]: \d{1, }

**Review by jonaqorri — CHANGES_REQUESTED** [review]

File: gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java (hunk 2 above)
Comment (msaperst) [reply gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java]: I thought we said 4 or more

Comment (jeffvoight) [reply gherkin-checks/src/main/java/org/sonar/gherkin/checks/FileNameCheck.java]: Yup. 4 and more is what we're going with.

File: gherkin-checks/src/main/java/org/sonar/gherkin/checks/OneSingleThenPerScenarioCheck.java (hunk 3 above)
Comment (jeffvoight) [reply gherkin-checks/src/main/java/org/sonar/gherkin/checks/OneSingleThenPerScenarioCheck.java:38]: It's in the doc, somewhere.

File: gherkin-checks/src/test/java/org/sonar/gherkin/checks/FileNameCheckTest.java (hunk 4 above)
Comment (jeffvoight) [reply gherkin-checks/src/test/java/org/sonar/gherkin/checks/FileNameCheckTest.java]: 4 or more.

**Review by georgiydeliya — APPROVED** [review]
lgtm

**Pull request merged**